### PR TITLE
Implement all required features

### PR DESCRIPTION
**Pull request by afizsavage**
- Connect to API
- Setup Redux
- Display list of countries with recorded covid 19 cases
- Filter list of countries by name
- Display complete details of recorded cases by country
- Add unit and integration test
- Add styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+
+## Technologies
+
+- React Bootstrap
+- React
+- Redux
+
+## Live Demo
+
+[Live Project](https://open-pandemic.herokuapp.com/)
```
Comment (omar-labana) [inline README.md:22]: - [ ] The deployed version isn't working properly, kindly check these console errors.
image

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import './App.css';
+import Header from './components/Header';
+
+const App = () => (
+  <>
+    <Header />
+    <Routes>
+      <Route exact path="/" element={<Countries />} />
+      <Route path="/details" element={<Details />} />
```
Comment (omar-labana) [inline src/App.js]: - [ ] According to the requirements, each category item detail page should have its unique route within the SPA. I'd suggest learning how to use URL parameters that react router provides.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @afizsavage, 

Excellent work so far! 👌
You have implemented all of the technical required features correctly. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+const Header = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  return (
+    <header className="b-primary flex white-text center-y space-between">
+      {location.pathname === '/' && <SearchBar />}
+      {location.pathname.includes('/details') && <BiArrowBack onClick={() => navigate('/')} />}
+      <span className="head-text">{location.pathname === '/' ? 'Cases' : 'Cases by country'}</span>
```
Comment (mmsesay) [inline src/components/Header/index.js:16]: - [OPTIONAL] I suggest you set this as a link to redirect the user to the homepage.

**Review by mmsesay — APPROVED** [review]
Hi @afizsavage, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**